env.example

```diff
@@ -2,4 +2,6 @@ REPORT_A_PROBLEM_ENABLED= # an undefined value means it is not enabled
 NOTIFY_BASE_API_URL=https://api.notification.canada.ca
 NOTIFY_REPORT_A_PROBLEM_EMAIL=experience@servicecanada.gc.ca
 NOTIFY_API_KEY=
-NOTIFY_REPORT_A_PROBLEM_TEMPLATE_ID=
\ No newline at end of file
+NOTIFY_REPORT_A_PROBLEM_TEMPLATE_ID=
+
+ADOBE_ENABLED = # true means content is taken from AEM api, false means content is taken from local file
```

Comment:
Missed this earlier. 

- Could we rename this file to `.env.example`? (It's missing the leading `.`).
- Also, the second half of your comment is potentially misleading. If you add a value of "false", it will show up in the system as `"false"`, the string, which would count as a truthy value. We should leave it undefined if we want a falsey value (as you currently have it).

Reply:
done